CI/before_install/linux_qt5.sh

```diff
@@ -9,6 +8,6 @@ sudo apt-get update
 # - debian build settings at debian/control
 sudo apt-get install libboost-dev libboost-filesystem-dev libboost-system-dev libboost-thread-dev libboost-program-options-dev libboost-locale-dev libboost-iostreams-dev \
 libsdl2-dev libsdl2-image-dev libsdl2-mixer-dev libsdl2-ttf-dev \
-qtbase5-dev \
+qtbase5-dev qttools5-dev \
 ninja-build zlib1g-dev libavformat-dev libswscale-dev libtbb-dev libluajit-5.1-dev \
-libminizip-dev libfuzzylite-dev qttools5-dev libsqlite3-dev # Optional dependencies
+libminizip-dev libfuzzylite-dev libsqlite3-dev # Optional dependencies
```

Comment:
Linux Qt 5/6 scripts should also be merged at some point to avoid copy-paste

Reply:
Or perhaps drop qt5 and always use qt6 for these CI's? They don't produce any artifacts and exists only to run tests & check different compilers. All conan-based CI's use qt5 so any qt5-specific issues will still be found

Reply:
don't think it would hurt to keep Linux Qt 5 as a sanity check

Reply:
Probably keeping as it for now.
One option is to install both qt5 and qt6, but then we'll need to specify which one to use in CMake
Another option is to split this script into common and qt-specific parts, but I'd rather keep things as simple as possible, without scripts calling other scripts.

If somebody else wants to change it - go ahead.